test/functional/tool_signet_miner.py

```diff
@@ -79,6 +80,38 @@ def mine_block(self, node):
             ], check=True, stderr=subprocess.STDOUT)
         assert_equal(node.getblockcount(), n_blocks + 1)
 
+    # generate block using the signet miner tool genpsbt and solvepsbt commands
+    def mine_block_manual(self, node, sign):
```

Comment:
nit: could enforce `sign` as keyword argument in order to enhance readability at the call-sites below
```suggestion
    def mine_block_manual(self, node, *, sign):
```